vscode-antimony/src/server/stibium/stibium/utils.py

```diff
@@ -62,3 +63,23 @@ def formatted_code(node: Optional[TreeNode]):
 
 def to_uri(path: str) -> str:
     return pathlib.Path(path).as_uri()
+
+def get_file_info(file: str):
+    antimony.clearPreviousLoads()
+    antimony.freeAll()
+    isfile = os.path.isfile(os.path.abspath(file))
+    if isfile:
+        from .api import AntFile
+        #sbml_file = antimony.loadSBMLFile(file)

```

Comment:
Does this mean that you've nixed importing SBML files for now?  Was something not working?

Reply:
Yeah, there's an issue with how the files are parsed when using getAntimonyString vs reading the files directly (an example is finding out whether declarations were constant or not)